### ISSUE TITLE
Hover Jquery error on Lookup

### ISSUE THREAD

**Issue by jmfarber** [alexed1/LightningFlowComponents#4]
Got the following error - Uncaught Action failed: c:LightningLookup$controller$hover [$ is not defined]

hover()@https://securis.lightning.force.com/components/c/LightningLookup.js:59:3

I'm using the installed package so maybe you've already fixed this.

**Comment by datapharmer** [comment]
I also get this error when using LookupFSC:

This page has an error. You might just need to refresh it. Action failed: c:LightningLookup$controller$hover [$ is not defined] Failing descriptor: {c:LightningLookup$controller$hover}

**Comment by datapharmer** [comment]
This issue appears to be cors related. The static resources are being served from .visual.force.com whereas the flow itself is being loaded from .salesforce.com and even if the origin is whitelisted there is no CORS header being served by Salesforce for the flows.

As a workaround, setting the API version for both the LightningLookup and LookupFSC to version 39 seems to workaround the problem, albeit less than an ideal fix.

My guess is this issue doesn't apply for those that have the pilot enabled for flow action components.

**Comment by alexed1** [comment]
The original reports of this were actually due to something really simple: jquery was not added to the package as a static resource and wasn't being installed at all.

We updated the package and for most people this seems to be working.

datapharmer, can you verify that you're still getting this error using the package available at https://sites.google.com/view/flowunofficial/flow-screen-components/lookup ? 

The pills function is unrelated: we never got it working and I didn't have time to dig into the base lookup component.

**Comment by allisonletts** [comment]
When I attempt to install from the Google Sites package into a sandbox, I get an error that the package was not found. So I downloaded the package from here and replaced the LookupFSC component that was having the issue. I'm still getting the error.

**Comment by alexed1** [comment]
allison, try this latest package https://login.salesforce.com/packaging/installPackage.apexp?p0=04tB0000000JIrm

**Comment by allisonletts** [comment]
@alexed1 I copied that link, but changed login. to test. because I'm installing in the sandbox. Same error, screenshotted below.
![image](https://user-images.githubusercontent.com/17914696/38825668-18742a46-4173-11e8-9900-2d814eae7db3.png)

**Comment by allisonletts** [comment]
@alexed1 I was able to install using GearSet. The problem seems to be resolved.

**Comment by alexed1** [comment]
There's a new version available at  https://sites.google.com/view/flowunofficial/flow-screen-components/lookup that addresses some of the issues described here.

**Comment by spilzer** [comment]
I have the lookupComponentFSC package v. 1.2 installed in my Sandbox org but am still getting the "JQuery hover" error. I have not yet tested it in Production to see if it is related to the org being a Sandbox but I don't think that should make a difference anyway, right? I'd love to be able to use the lookup component in my flows but can't seem to get past this bug!

**Comment by alexed1** [comment]
this is very frustrating. @ericrsmith35 can you take another look when you get a chance?

**Comment by alexed1** [comment]
I think the latest cause of the problem may be that when we made it a managed package it applied a namespace to the jquery staticresource, changing its name. However the code was looking for the old name.

I've created a new version with an unmanaged package at https://login.salesforce.com/packaging/installPackage.apexp?p0=04tB00000006Gl1

Try it and let me know if it solves the problems. I recommend that you first uninstall the old one if you have it installed.
